java/src/main/native/src/TableJni.cpp

```diff
@@ -859,13 +859,18 @@ JNIEXPORT long JNICALL Java_ai_rapids_cudf_Table_writeParquetBufferBegin(
     std::unique_ptr<cudf::jni::jni_writer_data_sink> data_sink(
         new cudf::jni::jni_writer_data_sink(env, consumer));
     sink_info sink{data_sink.get()};
-    chunked_parquet_writer_options opts =
+    cudf::jni::native_jintArray precisions(env, j_precisions);
+    std::vector<uint8_t> const v_precisions(
+        precisions.data(), precisions.data() + precisions.size());
+   chunked_parquet_writer_options opts =
```

Comment:
Indentation is off here. 
```suggestion
    chunked_parquet_writer_options opts =
```